GitTfs/Core/GitRepository.cs

```diff
@@ -155,7 +155,7 @@ private IGitTfsRemote ReadTfsRemote(string tfsUrl, string tfsRepositoryPath)
             }
         }
 
-        public IEnumerable<string> GetGitRemoteBranches(string gitRemote)
+        public IEnumerable<string> GetGitRepositoryBranchRemotes(string gitRemote)
```

Comment:
Not sure I like this rename, too... we are in the repository so no need to add it in the name.